packages/rx_bloc_cli/lib/src/extensions/xml_extensions.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:collection/collection.dart';
```

Comment:
Please confirm this is not copy/pasted code from some external library

Reply:
I can confirm that no code has been copy/pasted from outside sources in this PR. However, inspiration has been taken with necessary modifications where due.